.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 8.16.0
```

Comment:
What is this required for?

Reply:
@Haroenv 
Oh sorry...
I use [asdf](https://github.com/asdf-vm/asdf).
Should I remove that?

Reply:
I'd say yes, because https://github.com/algolia/shipjs/blob/master/.nvmrc is already set :)

Reply:
Done!